pandas/io/parsers.py

```diff
@@ -1133,6 +1151,7 @@ def __init__(self, src, **kwds):
 
         # XXX
         self.usecols = self._reader.usecols
+        self._usecols_dtype = _validate_usecols_arg(self.usecols)
```

Comment:
I don't think you are actually using this variable? ``_usecols_dtype``, so no need to save it (I think we discussed using it, but not necessary i  guess)